manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Automatically updated by the winget bot at 2021/Jul/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: 15.0.18386.0
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PublisherUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+Author: Microsoft
+PackageName: Microsoft SQL Server Management Studio
+PackageUrl: https://docs.microsoft.com/sql/ssms/download-sql-server-management-studio-ssms
+License: Copyright (c) Microsoft Corporation
+Copyright: Copyright (c) Microsoft Corporation
+ShortDescription: Microsoft.SQLServerManagementStudio
+Description: SQL Server Management Studio (SSMS) is an integrated environment for managing any SQL infrastructure, from SQL Server to Azure SQL Database. SSMS provides tools to configure, monitor, and administer instances of SQL Server and databases. Use SSMS to deploy, monitor, and upgrade the data-tier components used by your applications, and build queries and scripts.
+Tags:
+  - SQL
+  - SQL Server
+  - SQL Management Studio
+  - SSMS
+  - Management Studio
+Moniker: SSMS
```

Someone gonna correct this sooner or later
```suggestion
  - sql
  - sql-server
  - sql-management-studio
  - ssms
  - management-studio
Moniker: ssms
```